### PR TITLE
DOMException: Add test for Error.isError

### PR DESCRIPTION
**Pull request by lionel-rowe**
Supersedes https://github.com/tc39/test262/pull/4322

### CODE REVIEW

File: webidl/ecmascript-binding/es-exceptions/DOMException-is-error.any.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+test(function() {
+  // https://github.com/tc39/proposal-is-error/issues/9
+  // https://github.com/whatwg/webidl/pull/1421
+  if (typeof Error.isError === 'function') {
```
Comment (domenic) [inline webidl/ecmascript-binding/es-exceptions/DOMException-is-error.any.js:6]: This should not be conditional.

**Review by domenic — CHANGES_REQUESTED** [review]

**Review by domenic — APPROVED** [review]
LGTM. When https://github.com/whatwg/webidl/pull/1421 is ready to merge, we can merge this.

File: webidl/ecmascript-binding/es-exceptions/DOMException-is-error.any.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+'use strict';
```
Comment (ptomato) [inline webidl/ecmascript-binding/es-exceptions/DOMException-is-error.any.js:1]: Consider adding this:

```suggestion
// META: global=window,dedicatedworker,shadowrealm

'use strict';
```

(see #49159)

**Comment by syg** [comment]
This should be good to merge, yeah? Would like to see these synced up to chromium so I don't have to write my own tests.

**Pull request merged**